sysconfig.btrfsmaintenance

```diff
@@ -15,10 +15,10 @@ BTRFS_AC_POWER_TIMEOUT=3600
 
 ## Path:        System/File systems/btrfs
 ## Type:        string
-## Default:     "/sys/class/power_supply/AC/online"
+## Default:     "/sys/class/power_supply/*/online"
 #
 # Path of AC status flag
-BTRFS_AC_POWER_DEVICE="/sys/class/power_supply/AC/online"
+BTRFS_AC_POWER_DEVICE="/sys/class/power_supply/*/online"
```

Comment:
This looks like it will do the trick for laptops.  I mentioned upower in another comment, because I think that /sys/class/power_supply/*/online will return true even when a server is on UPS battery. (eg: NUT or apcupsd).  IIRC it provides a generic interface, and alternatives are `upsmon` (NUT) and `apcaccess -p TONBATT` (apcupsd)

Reply:
We may want to revisit that later, with support for UPS, but for now I'm fine with the more relaxed check. Meanwhile I verified the path exists on my other systems, so I think there's nothing more left to discuss.

Reply:
I had no time to dedicate to search a portable solution yet.